### PR TITLE
Closes #112: Rewrite document-classification transformer to spark

### LINKED ISSUE #112: Rewrite document-classification transformer to spark

### PR DESCRIPTION
**Pull request by lukdumi**

### CODE REVIEW

File: iis-workflows/iis-workflows-top/src/main/resources/eu/dnetlib/iis/workflows/top/primary/processing/oozie_app/workflow.xml [hunk 1]
```diff
@@ -711,48 +711,26 @@
...
 			<default to="skip-documentsclassification" />
 		</switch>
 	</decision>
 
-	<action name="transformers_documentsclassification">
+	
+	<action name="documentsclassification">
 		<sub-workflow>
-			<app-path>${wf:appPath()}/transformers_documentsclassification</app-path>
+			<app-path>${wf:appPath()}/documentsclassification</app-path>
```
Comment (madryk) [inline iis-workflows/iis-workflows-top/src/main/resources/eu/dnetlib/iis/workflows/top/primary/processing/oozie_app/workflow.xml]: You could remove priming for old transformer workflow in import.txt.
And definitely you should modify documentsclassification_main workflow to documentsclassification as you changed it:
`documentsclassification_main classpath eu/dnetlib/iis/workflows/documentsclassification/main/oozie_app`

File: pom.xml [hunk 2]
```diff
@@ -939,6 +939,15 @@
                     </execution>
                 </executions>
             </plugin>
+            
+             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-resources-plugin</artifactId>
+                    <version>2.7</version>
+                    <configuration>
+                        <encoding>UTF-8</encoding>
```
Comment (madryk) [inline pom.xml]: maybe `<encoding>${project.build.sourceEncoding}</encoding>`?
It's weird that you had to define it. Documentation states that ${project.build.sourceEncoding} is default value for encoding in this plugin

File: iis-workflows/iis-workflows-top/src/main/resources/eu/dnetlib/iis/workflows/top/primary/processing/oozie_app/workflow.xml (hunk 1 above)
Comment (lukdumi) [inline iis-workflows/iis-workflows-top/src/main/resources/eu/dnetlib/iis/workflows/top/primary/processing/oozie_app/workflow.xml]: A good remark :+1:

File: pom.xml (hunk 2 above)
Comment (lukdumi) [inline pom.xml]: Yes, it was my desperation ;-)

**Pull request closed without merging**